dashboard/client/index.js

```diff
@@ -41,8 +41,18 @@ Template.dashboard.onCreated(function () {
     }
   }
 
-  instance.getAggrData = opts => new Promise((resolve, reject) => {
-    Meteor.call('getAggr', opts, (err, res) => {
+  instance.getChartData = opts => new Promise((resolve, reject) => {
+    Meteor.call('getAggregatedData', opts, (err, res) => {
+      if (err) reject(err)
+      Meteor.call('parseDataForNvd', res, (err1, res1) => {
```

Comment:
Should this be a Meteor method, or just a client side funciton? I am asking, be cause Meteor methods are mainly server-side (with client side stub). That means, in this scenario, we are getting aggregated data from the server and then sending it back to the server to be parsed and returned again.